src/adapter/DatabaseAdapter.php

```diff
@@ -57,7 +57,7 @@ public function savePolicyLine($ptype, array $rule)
      */
     public function loadPolicy(Model $model): void
     {
-        $rows = $this->model->cache('tauthz')->select()->toArray();
+        $rows = $this->model->cache(config('tauthz.enforcers.basic.database.cahce_key', 'tauthz'))->select()->toArray();

```

Comment:
If there are multiple enforcers, you need to distinguish the configuration of which enforcer to read, please add it to TauthzService.

Reply:
If there are multiple enforcers,You can set the cache's prefix of the thinkphp

Reply:
Each group of configuration corresponds to an enforcer, the default is the `basic` group, but you cannot fix it as `basic`.